src/main/java/org/jitsi/videobridge/AbstractEndpointMessageTransport.java

```diff
@@ -479,7 +499,7 @@ protected void onReceiverVideoConstraintEvent(
         }
 
         videoConstraintsCompatibility.setMaxFrameHeight(maxFrameHeight);
-        onVideoConstraintsChangedEvent(videoConstraintsCompatibility.computeVideoConstraints());
+        senderVideoConstraintsChangedEvent(videoConstraintsCompatibility.computeVideoConstraints());
```

Comment:
Does that mean that we have two completely independent code paths that send the "sender video constraints" message -- one from selected/pinned/old receiver constraint events (that goes through the compatibility layer), and one from the new "receiver constraints" message (which does NOT go through the compatibility layer)?

Reply:
Yes

Reply:
Reading further, support for the new message is a no-op. Are you *planning* to make it go through the compatibility layer?

Reply:
The method that this comment pertains to got renamed to `receiverVideoConstraintsChangedEvent ` but I suppose you're still talking about `COLIBRI_CLASS_RECEIVER_VIDEO_CONSTRAINTS_CHANGED`.

I wasn't planning to add support for the message in this PR because I don't want to delay things any further and it's not functionally necessary; I'm happy to work on it if you think it's better to have it as part of this PR. Its main usage will be for the octo part, at least that was how it was setup in the previous PR.